README.md

```diff
@@ -121,7 +121,7 @@ Examples: see [example/angular1](https://github.com/google/dart-scissors/tree/ma
   ```
 
 Valid settings:
-- `pruneCss` (boolean): by default, `true` in `release` only
+- `pruneCss` (`skip` or `print` or `overwrite`): by default, `overwrite` in `release` only
```

Comment:
Thanks Olivier. 
Can you please point me to existing usage of pruneCss. Because AFAIS I have replaced all the usages. So this should work fine.